src/compas_fab/ghpython/components/Cf_PlanCartesianMotion/code.py

```diff
@@ -23,8 +24,9 @@ def RunScript(
 
         if robot and robot.client and robot.client.is_connected and start_configuration and planes and compute:
             frames = [plane_to_compas_frame(plane) for plane in planes]
+            frame_waypoints = FrameWaypoints(frames)
```

Comment:
This got me thinking: this change is ok for this component, but how are we going to allow point-axis planning in Grasshopper, considering there's no point&axis entity that can be passed around? This is also evidenced by the fact that `PointAxisWaypoints` takes a list of tuples of points and vectors. So I first thought it could be a good idea to just pass a list of frames, and have a boolean, if that is set to true, we reduce the frame to a point&axis in the component. But there I started questioning if we need at all a `PointAxisWaypoints` class, or if we can instead have only `FrameWaypoints` containing that bool as part of its own data? It could be named something technical like `Underconstrained=True`, or something more humanly readable like `AllowRotationAroundNormal=True`. And since the Axis part of PointAxis is limited to the normal of the frame, I am currently leaning towards going with something more like the latter. What do you think?

Reply:
Although I'm not 100% convinced that `PointAxis` still makes sense, I don't have a solid alternative to provide so, we'll keep it.